src/elements/line.ts

```diff
@@ -15,18 +15,16 @@ const DASH_VALUES = [
 export default class LineElement {
     size
     color
-    dash
+    dash = 'solid'
```

Comment:
You should not put a default value - only check that the value is valid (and in the case it is not, remove it)

A console.warn if the value is not correct would be nice

Reply:
not sure I follow the logic, in the original code this effectively was achieved here https://github.com/sgenoud/bulletpoints/pull/2/files#diff-dd4b6ee7080fd5895a48971555ab9e33L26 so I simply moved the default into the declaration.

you prefer to avoid the default value at all? (like with `cap`)

Reply:
yeah I would prefer for it to behave like `cap`! Sorry I was a bit messy!

Reply:
roger that! the commit is pushed and works fine according to my tests.

regarding the console.warn, I don't think it's the proper way in this case. 

I think the proper solution requires more time investment and is either of two:
1) throw exception – not ideal, but does the job
2) define dash as an [Enum](https://www.typescriptlang.org/docs/handbook/enums.html) and specify as the type in the method param accordingly.